apps/client/src/components/compositions/AccountSummarySection/request.ts

```diff
@@ -1,9 +1,29 @@
 import { useQuery } from 'react-query'
 import request from '../../../api'
 
+interface RecordI {
+  id: string
+  balance: number
+}
+interface WalletI {
+  id: string
+  balance: number
+  records?: RecordI[]
+}
+interface WalletDataI {
+  [key: string]: WalletI
+}
+
+interface DataI {
+  data: WalletDataI
+}
```

Comment:
You type don't match the response

Reply:
<img width="584" alt="image" src="https://github.com/Ajo-Savings-Project/ajo-savings/assets/48376769/89f23e33-52df-4df6-aa25-65792835a3ef">


I expect something like this

<img width="300" alt="image" src="https://github.com/Ajo-Savings-Project/ajo-savings/assets/48376769/4472b70b-1f6c-4bbd-9c4e-28fc432affa3">

But, you'll be writing a zod schema instead.